languages/ruby/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,104 @@
+In Ruby, **arrays** are ordered, integer-indexed collections of any object. Array indexing starts at 0, as in C or Java. A negative index is assumed to be relative to the end of the array—that is, an index of -1 indicates the last element of the array, -2 is the next to last element in the array, and so on.
+Ruby arrays mixed in the [Enumerable module][enumerable-module] which provides collection classes with several traversal and searching methods, and with the ability to sort.
+
+### Create array.
+
+- An array in Ruby can contain different types of objects.
+
+```ruby
+array = [1, "two", 3.0] #=> [1, "two", 3.0]
+```
+
+- An array can also be created by explicitly calling ::new
+
+```ruby
+array = Array.new #=> []
+Array.new(3)      #=> [nil, nil, nil]
+```
+
+### Element Assignment
+
+- Sets the element at index, or replaces a subarray from the start index for length elements.
+
+```ruby
+a = Array.new
+
+a[4] = "4"                  #=> [nil, nil, nil, nil, "4"]
+a[0, 3] = [ 'a', 'b', 'c' ] #=> ["a", "b", "c", nil, "4"]
+```
+
+- Negative indices will count backward from the end of the array.
+
+```ruby
+a = ['a, 'b']
+
+a[-1] = "Z" #=> ["a", "Z"]
```

Comment:
```suggestion
a[-1] = "Z" 
a #=> ["a", "Z"]
```

The first line actually returns `"Z"` so we should have a second line printing the array.